go/ql/lib/semmle/go/security/LogInjection.qll

```diff
@@ -15,6 +15,8 @@ module LogInjection {
   import LogInjectionCustomizations::LogInjection
 
   /**
+   * DEPRECATED: Use `Flow` instead.
+   *
```

Comment:
Can't comment on the relevant line because it wasn't changed as part of this PR, but `Flow` here doesn't have any documentation.